### DISCUSSION TITLE
Is there a complete tutorial for how to create a plugin using go-client?

### DISCUSSION THREAD

**Discussion by alwindoss** [neovim/neovim#33029] [Q&A] [answered]
There is an example on the README.md file, but that is just an example, it does not go into the details of what is happening in there. A description would help and certainly would go a long way in helping a noob get upto speed.

**Comment by glepnir** [comment, marked as answer]
a simple usage example https://github.com/WhoIsSethDaniel/nvim-go-client-examples
